source/drivers/php-libraries.txt

```diff
@@ -60,6 +62,8 @@ Framework Integrations
     library and includes an abstract model, a helper class for handling bulk
     writes, database wrappers for Laravel and Lumen, and some necessary helper
     functions.
+
+  - `Mongolid Laravel <https://github.com/leroy-merlin-br/mongolid-laravel>`_: Easy, powerful and ultrafast MongoDB ODM for Laravel.
```

Comment:
```suggestion
  - `Mongolid Laravel <https://github.com/leroy-merlin-br/mongolid-laravel>`_:
    Laravel package for the MongoLid ODM. Includes integration for the auth
    and queue failed job components.
```

I don't think this needs to repeat the marketing description for the main library. It'd be more helpful to note what specific integrations it provides. AFAICT, this doesn't have anything to do with Eloquent (unlike the jenssegers library).